package.json

```diff
@@ -34,7 +34,7 @@
     "@octokit/plugin-throttling": "9.3.1",
     "@octokit/rest": "21.0.2",
     "@octokit/types": "13.5.0",
-    "testing-farm": "^1.9.2",
+    "testing-farm": "1.10.0",
```

Comment:
I don't follow this. Sometimes we have `^1.9.2` and sometimes `1.10.0`. I would prefer to keep format.

Reply:
This is my fault. I changed it in the last PR and didn't notice before merging. I prefer the format without `^`.